includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -224,6 +224,23 @@ public function get_intent( $intent_id ) {
 		return $this->deserialize_intention_object_from_array( $intent );
 	}
 
+	/**
+	 * Create a payment intent.
+	 *
+	 * @param array $request data about the intent.
+	 *
+	 * @return array Data of the created intent.
+	 *
+	 * @throws API_Exception Error creating intent.
+	 */
+	public function create_intent( $request ) {
+		return $this->request(
+			$request->get_params(),
+			self::INTENTIONS_API,
+			self::POST
+		);
+	}
```

Comment:
IMHO, we have already had `Create_Intention` core request class for the same purpose, we would not introduce this method? 

https://github.com/Automattic/woocommerce-payments/blob/08bd8a0d5a60e027ee155148e7f64b966234705b/includes/core/server/request/class-create-intention.php#L18